admin/AmazonAI-GeneralConfiguration.php

```diff
@@ -197,22 +212,38 @@ public function posttypes_gui() {
     function tim_limitless_gui(){
         if($this->common->is_tim_limitless_enabled()){
             $checked                = ' checked ';
-            $trinity_connected      = true;
         }else{
             $checked                = ' ';
+        }
+        echo '<input style="display:none;" type="checkbox" name="'.TIM_LIMITLESS_ENABLED.'" id="'.TIM_LIMITLESS_ENABLED.'" ' . esc_attr($checked) .'> <p class="description"></p>';
+    }
+
+    function tim_limitless_credentials_separator_gui(){
+        //Empty;
+    }
+
+    function amazon_polly_credentials_radio_gui(){
+        //EMPTY
+    }
+
+    function tim_limitless_credentials_radio_gui(){
+        if($this->common->is_tim_limitless_enabled()){
```

Comment:
`is_tim_limitless_enabled` => `trinity_connected` 
make the renames as you're touching the code, no need to do it in a separate move